docs/open/QUICKSTART.md

```diff
@@ -1,63 +1,89 @@
-If you're on a Mac environment with homebrew installed, run this command.
+## Installing Terraform.
+If you're on a Mac environment with homebrew installed, simply run the following command:
 ```bash
 brew install terraform
 ```
 
-For help installing Terraform on a different OS, see [here](https://www.terraform.io/downloads.html):
+Once this command completes, you should be able to run the following command and see output consistent with the version of Terraform you have installed:
+```bash
+$ terraform version
+Terraform v0.11.8
+```
+
+For help installing Terraform on a different OS, please see [here](https://www.terraform.io/downloads.html):
 
+## Ensure you have your AWS Cloud Credentials Properly Set up
+I'm not sure exactly waht the best way to do this is. 
+Someone should fill this in with more instructions.
 
-## Ensure AWS Default Region
-Current Terraform AWS Provider requires that the default region variable be set. You can set the default region using the following command:
+## Set the Default AWS Region
+The current Terraform Provider for AWS requires that the default AWS region be set before it can be used. You can set the default region with the following command:
 ```bash
-export AWS_DEFAULT_REGION="desired-aws-region"
+export AWS_DEFAULT_REGION="<desired-aws-region>"
 ```
-Example:
+For Example:
 ```bash
 export AWS_DEFAULT_REGION="us-east-1"
 ```
 
-Ensure it is set:
+Ensure it has been set:
 ```bash
 > echo $AWS_DEFAULT_REGION
 us-east-1
 ```
 
-## Add your keys to your ssh agent:
+## Add your SSH keys to your ssh agent:
+
+Terraform requires SSH access to the instances you launch as part of your DC/OS cluster. As such, we need to make sure that the SSH key used to SSH to these instances is added to your `ssh-agent`, prior to running `terraform`.
+
+If you need help on creating an SSH key-pair for AWS prior to running the command below, please follow the instructions [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html).
+
+Otherwise, just run the following command to add your key to the `ssh-agent`:
 
 ```bash
-ssh-add <path_to_your_private_ssh_key>
+ssh-add <path_to_your_private_aws_ssh_key>
 ```
 
+For Example:
+```bash
+ssh-add ~/.ssh/aws-id-rsa
+```
 
 # Creating a Cluster
 
-1) Let’s start by creating a local folder.
+1) Let’s start by creating a local folder and cd'ing into it. This folder will be used as the staging ground for downloading all required Terraform modules and holding the configuration for the cluster you are about to create. 
 
 ```bash
 mkdir dcos-tf-aws-demo && cd dcos-tf-aws-demo 
 ```
 
-2) Copy and paste the example code below into a new file and save it as `main.tf` in our folder.
+2) Once that is done, copy and paste the example code below into a new file and save it as `main.tf` in the newly created folder.
 
-The example code below creates a DC/OS OSS 1.11.4 cluster on AWS with:
+This example code tells Terraform to create a DC/OS OSS 1.11.4 cluster on AWS with:
```

Comment:
How do we know it installs 1.11.4 (and how do we ensure this example code does this forever going forward). There is nothing in the `main.tf` that specifies this.

Reply:
correct. We should state to better fix the version. It will install whatever we state is latest but for usability we should suggest fixing the version

Reply:
we should place an explicit version